styles/index.css

```diff
@@ -128,9 +129,36 @@ body {
   border-radius: 7px;
   font-size: 20px;
   font-family: sans-serif;
+  background-color: thistle;
 }
 .skills {
   display: flex;
   margin: 20px;
   font-size: 25px;
 }
+.home {
+  color: white;
+}
+.bookmark {
+  color: white;
+}
+.paper {
+  color: white;
+}
+.person {
+  color: white;
+}
```

Comment:
Unnecessary duplication – just make one class `text-white` and use it everywhere.